### ISSUE TITLE
Add a chrome plugin

### ISSUE THREAD

**Issue by vaidik** [vinayak-mehta/itslit#4]
So that every new tab can prompt a new lighting talk on new tabs or may be once every day or something like that?

**Comment by vinayak-mehta** [comment]
> So that every new tab can prompt a new lighting talk on new tabs

Do you mean something like https://github.com/planetlabs/planet-view and https://brainytab.com/ that recommend a new thing whenever you open a tab? That is a good idea, but out of scope for the Python CLI tool. I might try to make a Chrome extension at some point!

> or may be once every day or something like that?

I do this with cron. I've selected a time when I'll definitely be in front of a computer so that itslit opens a new tab in an existing window. Though, that can sometimes be a annoyance when you're working :sweat_smile:

**Comment by vaidik** [comment]
Yeah I thought about cron myself and opened this issue exactly for the reason you mentioned. I think there is a pretty solid UX pattern today around random discovery of information - which is the new Chrome tab pattern.

I'm sure a lot of folks would really like that pattern. I know I would :) 

How to do it is up to you? I guess calling a Python CLI from a chrome plugin won't be possible. I am not sure if there is a way to run Python in the browser? WASM perhaps?

**Comment by vinayak-mehta** [comment]
This could totally be done with WASM! I rewrote itslit in Rust: https://github.com/vinayak-mehta/itslit-rs which should be able to be compiled into WASM: https://vinayak.io/2020/09/06/day-20-make-js-run-faster-with-this-simple-trick which can then be bundled with a Chrome extension!

I might try picking this up in the future. If you're interested, you can also try doing this!

**Comment by vinayak-mehta** [comment]
Though it might just be easier to write the Chrome extension with vanilla Javascript :sweat_smile:
